src/identity/src/mic.js

```diff
@@ -146,7 +133,7 @@ export class MobileIdentityConnect extends Identity {
               redirected = true;
               popup.removeAllListeners();
               popup.close();
-              resolve(url.parse(event.url, true).query.code);
+              resolve(qs.parse(event.url.split('?')[1]).code);
```

Comment:
@thomasconner the `url.split` may not have 2 (or more) parts. This code needs to be more defensive to account for a url with no query.